### PR TITLE
Api review changes

### PR DESCRIPTION
**Pull request by kichalla**
@Eilon 

I will post an announcement once this and reaction changes go in.

### CODE REVIEW

File: src/Microsoft.AspNet.Cors/CorsMiddleware.cs [hunk 1]
```diff
@@ -4,7 +4,7 @@
 using System;
 using System.Threading.Tasks;
 using Microsoft.AspNet.Builder;
-using Microsoft.AspNet.Cors.Core;
+using Microsoft.AspNet.Cors.Infrastructure;
 using Microsoft.AspNet.Http;
 using Microsoft.Extensions.Primitives;
```
Comment (Eilon) [inline src/Microsoft.AspNet.Cors/CorsMiddleware.cs]: I believe this type is supposed to move to `.Infrastructure`

Comment (kichalla) [inline src/Microsoft.AspNet.Cors/CorsMiddleware.cs]: good catch...you are right...also the CorsPolicyBuilder I think...

Comment (Eilon) [inline src/Microsoft.AspNet.Cors/CorsMiddleware.cs]: Yeah everything except the two attributes I think. It's in https://github.com/aspnet/CORS/issues/41.

**Comment by Eilon** [comment]
Other than that, :shipit:

**Comment by kichalla** [comment]
475771250bf4f6b8b03ee1c7d30c9f4e4b4816e8

**Comment by kichalla** [comment]
https://github.com/aspnet/CORS/issues/41

**Pull request closed without merging**